### PR TITLE
Fix opening repeater row when creating new one

### LINKED ISSUE #574: Repeater not properly opening when adding a new item
9.0.0beta7 docker image with DB pointing on a mysql 5.7

Issue: When adding a new item of a repeater, the repeater adds a new item in a closed state, and when clicking to open it, it opens and closes right away. Only the next click finally properly opens it.

Expected behavior: Adding a new item of a repeater adds the new item in an open state.

### PR DESCRIPTION
**Pull request by Nitwel**
closes #574

### CODE REVIEW

File: app/src/components/v-item-group/v-item.vue [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <template>
 	<div class="v-item">
-		<slot v-bind="{ active, toggle }" />
+		<slot v-bind="{ 'active': isActive, toggle }" />
```
Comment (rijkvanzanten) [inline app/src/components/v-item-group/v-item.vue:3]:
```suggestion
		<slot v-bind="{ active: isActive, toggle }" />
```

File: app/src/components/v-item-group/v-item.vue [hunk 2]
```diff
@@ -18,14 +18,25 @@ export default defineComponent({
...
 			default: 'item-group',
 		},
+		active: {
+			type: Boolean,
+			default: undefined
+		},
+		watch: {
+			type: Boolean,
+			default: true
+		}
```
Comment (rijkvanzanten) [inline app/src/components/v-item-group/v-item.vue:28]: What does watch do

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: app/src/components/v-item-group/v-item.vue (hunk 2 above)
Comment (Nitwel) [reply app/src/components/v-item-group/v-item.vue:28]: enables/disables the watcher. If set to false the `active ref<bool>` will not update the state of the item after it got initialized.

Comment (rijkvanzanten) [reply app/src/components/v-item-group/v-item.vue:28]: Could you elaborate on that? I don't know what you mean by 

> If set to false the active ref<bool> will not update the state of the item after it got initialized.

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**